tests/FsCodec.NewtonsoftJson.Tests/StreamTests.fs

```diff
@@ -1,23 +1,28 @@
 ﻿#if SYSTEM_TEXT_JSON
-module FsCodec.SytemTextJson.Tests.StreamTests
+module FsCodec.SystemTextJson.Tests.StreamTests
 open FsCodec.SystemTextJson
 #else
 module FsCodec.NewtonsoftJson.Tests.StreamTests
 open FsCodec.NewtonsoftJson
 #endif
 
-open Xunit
-open System.IO
 open Swensen.Unquote
-
-let serdes = Serdes Options.Default
+open System.IO
+open Xunit
 
 type Rec = { a : int; b : string; c : string }
 
 let [<Fact>] ``Can serialize/deserialize to stream`` () =
     let value = { a = 10; b = "10"; c = "" }
     use stream = new MemoryStream()
-    serdes.SerializeToStream(value, stream)
-    stream.Seek(0L, SeekOrigin.Begin) |> ignore
-    let value' = serdes.DeserializeFromStream(stream)
-    <@ value = value' @>
```

Comment:
@deviousasti this 'assertion' is failing on NSJ - any ideas why? (See altered version showing STJ vs NSJ discrepancies) (Pretty sure this code path is unaffected by any code moves in this PR)